### PR TITLE
431 command to kick the player from the game

### LINKED ISSUE #431: [Command] /kick - kick a player from the game

### PR DESCRIPTION
**Pull request by MarcusSilvaDev**

### CODE REVIEW

File: data/extensions/Spells/Commands/KickPlayerCommand.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            return false;
+        }
+
+        if (player is null || player.CreatureId == actor.CreatureId)
+        {
+            error = InvalidOperation.NotPossible;
+            return false;
+        }
+        
+        ctx.RemovePlayer(player);
```
Comment (caioavidal) [inline data/extensions/Spells/Commands/KickPlayerCommand.cs]: maybe using the Logout method from the Player is better.

File: src/Database/NeoServer.Data/Configurations/AccountModelConfiguration.cs [hunk 2]
```diff
@@ -86,6 +86,15 @@ private static void Seed(EntityTypeBuilder<AccountModel> builder)
                 Password = "1",
                 PremiumTime = 30,
                 AllowManyOnline = true
+            },
```
Comment (caioavidal) [inline src/Database/NeoServer.Data/Configurations/AccountModelConfiguration.cs]: why creating a second account for another GOD ?

File: src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs [hunk 3]
```diff
@@ -135,7 +135,7 @@ public static void Seed(EntityTypeBuilder<PlayerModel> builder)
             {
                 PlayerId = 4,
                 PlayerType = 1,
-                AccountId = 1,
+                AccountId = 2,
```
Comment (caioavidal) [inline src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs]: same here?

File: src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs [hunk 4]
```diff
@@ -177,9 +177,9 @@ public static void Seed(EntityTypeBuilder<PlayerModel> builder)
             {
                 PlayerId = 5,
                 PlayerType = 1,
-                AccountId = 1,
```
Comment (caioavidal) [inline src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs]: same here?

File: src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs [hunk 5]
```diff
@@ -177,9 +177,9 @@ public static void Seed(EntityTypeBuilder<PlayerModel> builder)
...
+                AccountId = 2,
                 TownId = 1,
-                Name = "Paladin Sample",
+                Name = "Paladin",
```
Comment (caioavidal) [inline src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs]: why removing sample from Paladin but not from other vocations? ie : Sorcerer Sample...

**Review by caioavidal — CHANGES_REQUESTED** [review]

File: src/Database/NeoServer.Data/Configurations/AccountModelConfiguration.cs (hunk 2 above)
Comment (MarcusSilvaDev) [reply src/Database/NeoServer.Data/Configurations/AccountModelConfiguration.cs]: I neeed test with multiple account for kick another player. 

Sample:
    Account 1[GOD] -> kick -> account 2

File: data/extensions/Spells/Commands/KickPlayerCommand.cs (hunk 1 above)
Comment (MarcusSilvaDev) [reply data/extensions/Spells/Commands/KickPlayerCommand.cs]: Does the method logout already remove context?

File: src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs (hunk 3 above)
Comment (MarcusSilvaDev) [reply src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs]: added multiple account.

File: src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs (hunk 4 above)
Comment (MarcusSilvaDev) [reply src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs]: added multiple account.

File: src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs (hunk 5 above)
Comment (MarcusSilvaDev) [reply src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs]: I wanted add multiple account.

File: data/extensions/Spells/Commands/KickPlayerCommand.cs (hunk 1 above)
Comment (caioavidal) [reply data/extensions/Spells/Commands/KickPlayerCommand.cs]: no. you can use the PlayerLogOutCommand instead. 
or do:
player.Logout(true)
creatureManager.RemovePlayer(player);

File: src/Database/NeoServer.Data/Configurations/AccountModelConfiguration.cs (hunk 2 above)
Comment (caioavidal) [reply src/Database/NeoServer.Data/Configurations/AccountModelConfiguration.cs]: ah okay. Can you remove these changes from the PR then?

**Review by caioavidal — APPROVED** [review]

**Pull request merged**